core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceServiceSpec.scala

```diff
@@ -1303,7 +1304,14 @@ class WorkspaceServiceSpec extends AnyFlatSpec with ScalatestRouteTest with Matc
     verify(services.googleAccessContextManagerDAO, Mockito.never()).overwriteProjectsInServicePerimeter(any[ServicePerimeterName], any[Set[String]])
   }
 
-  it should "claim a Google Project from Resource Buffering Service" in pending
+  it should "claim a Google Project from Resource Buffering Service" in withTestDataServices { services =>
+    val newWorkspaceName = "space_for_workin"
+    val workspaceRequest = WorkspaceRequest(testData.testProject1Name.value, newWorkspaceName, Map.empty)
+
+    val workspace = Await.result(services.workspaceService.createWorkspace(workspaceRequest), Duration.Inf)
+
+    verify(services.resourceBufferService, Mockito.atLeastOnce()).getGoogleProjectFromBuffer(any[ProjectPoolType], any[String])
```

Comment:
```suggestion
    verify(services.resourceBufferService).getGoogleProjectFromBuffer(any[ProjectPoolType], any[String])
```
Leaving the second parameter blank means it will check that the method was called exactly once. 

Of course same goes for the `cloneWorkspace` test

Reply:
cool, TIL!